Surface_mesh_parameterization/include/CGAL/Surface_mesh_parameterization/Orbifold_Tutte_parameterizer_3.h

```diff
@@ -100,7 +100,7 @@ enum Weight_type
 ///
 /// \cgalModels `Parameterizer_3`
 ///
-/// \tparam SeamMesh must be a `Seam_mesh`, with underlying mesh any model of `FaceGraph`
+/// \tparam SeamMesh must be a `Seam_mesh`, with underlying mesh any model of `FaceListGraph`
```

Comment:
Note that In general, if you need to iterate over halfedges, you must also request `HalfedgeListGraph`. This is a general remark not only in this context.